src/Symfony/Component/Notifier/Bridge/AllMySms/AllMySmsTransport.php

```diff
@@ -64,11 +64,11 @@ protected function doSend(MessageInterface $message): SentMessage
         $endpoint = sprintf('https://%s/sms/send/', $this->getEndpoint());
         $response = $this->client->request('POST', $endpoint, [
             'auth_basic' => $this->login.':'.$this->apiKey,
-            'body' => [
+            'body' => json_encode([
```

Comment:
could you try replacing `'body'` by `'json'` instead of adding `json_encode()` around?

Reply:
Yes it's work too. I have make the change.